sdk/storage/Azure.Storage.Blobs/swagger/readme.md

```diff
@@ -1046,6 +1046,8 @@ directive:
 ```
 
 ### Make lease duration a long
+### Since TimeSpan.MaxValue would overflow an int, we add a swagger transform to make the duration used in the internal generated BlobRestClient be a long. 
+### TimeSpan is used within LeaseClient, and is transformed to a long before passing off to BlobRestClient.

```

Comment:
Remove the `###`s before your comments.  That's to mark a section heading.

And maybe rewrite this to something a little more informative for those without context like `We're representing the duration as a TimeSpan in the .NET client libraries and TimeSpan.MaxValue would overflow an int so we're changing to a long.  Values larger than int.MaxValue are correctly parsed by the service.`